<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>587</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Транскрибатор аудиофайлов</string>
  </property>
  <property name="windowIcon">
   <iconset theme="C:\Users\Klim\Desktop\PYTHON\PracticeProject_1\bird2.ico" resource="resources.qrc">
    <normaloff>:/image/bird2.ico</normaloff>:/image/bird2.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QPushButton" name="searchButton">
      <property name="text">
       <string>Искать</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="searchEdit"/>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QTextEdit" name="outputEdit">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>856</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="openFile"/>
    <addaction name="saveFile"/>
    <addaction name="separator"/>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Конвертация</string>
    </property>
    <addaction name="m4a_to_wav"/>
    <addaction name="mp3_to_wav"/>
    <addaction name="flv_to_wav"/>
    <addaction name="ogg_to_wav"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Язык</string>
    </property>
    <addaction name="rus"/>
    <addaction name="eng"/>
   </widget>
   <widget class="QMenu" name="menuTelegram">
    <property name="title">
     <string>Telegram</string>
    </property>
    <addaction name="add_bot"/>
    <addaction name="show_ID"/>
    <addaction name="send"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="about"/>
    <addaction name="helpMe"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menuTelegram"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="openFile">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="saveFile">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="m4a_to_wav">
   <property name="text">
    <string>m4a to wav</string>
   </property>
  </action>
  <action name="rus">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Русский</string>
   </property>
  </action>
  <action name="eng">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Английский</string>
   </property>
  </action>
  <action name="mp3_to_wav">
   <property name="text">
    <string>mp3_to_wav</string>
   </property>
  </action>
  <action name="flv_to_wav">
   <property name="text">
    <string>flv_to_wav</string>
   </property>
  </action>
  <action name="ogg_to_wav">
   <property name="text">
    <string>ogg_to_wav</string>
   </property>
  </action>
  <action name="add_bot">
   <property name="text">
    <string>Подключить ID</string>
   </property>
  </action>
  <action name="about">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="helpMe">
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
  <action name="show_ID">
   <property name="text">
    <string>Показать ID</string>
   </property>
  </action>
  <action name="send">
   <property name="text">
    <string>Отправить </string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>473</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
